### PR TITLE
feat(fs-bigquery-export): Add firestore document ID to BigQuery change log table

### PR DESCRIPTION
**Pull request by Neilpoulin**
This addresses #345 and adds the firestore document ID to the big query change log table.

### CODE REVIEW

**Comment by Neilpoulin** [comment]
@googlebot I signed it!

**Comment by Neilpoulin** [comment]
@googlebot I signed it!

File: firestore-bigquery-export/scripts/gen-schema-view/src/__tests__/fixtures/sql/fullSchemaLatest.txt [hunk 1]
```diff
@@ -60,6 +64,24 @@ FROM
                 timestamp DESC
             )
           ) AS last_login,
+          SAFE_CAST(
```
Comment (russellwheatley) [inline firestore-bigquery-export/scripts/gen-schema-view/src/__tests__/fixtures/sql/fullSchemaLatest.txt:67]: These updates will have to be changed once this PR is merged in: https://github.com/firebase/extensions/pull/357

File: firestore-bigquery-export/functions/lib/util.js [hunk 2]
```diff
@@ -26,3 +27,10 @@ function getChangeType(change) {
     return firestore_bigquery_change_tracker_1.ChangeType.UPDATE;
 }
 exports.getChangeType = getChangeType;
+function getDocumentId(change) {
+    if (change.after.exists) {
```
[the remark this answers is not in the document]
Comment (Neilpoulin) [reply firestore-bigquery-export/functions/lib/util.js:31]: oops, yeah good call. line 34 should be `return change.before.id`. The thinking there is that if the document was deleted, `change.after` wouldn't exist so we should fall back to `change.before`. Likewise, if the document was just created, `change.after` wouldn't exist, so we should try to get the ID both ways. Will update.

**Comment by russellwheatley** [comment]
This PR has been broken up into the following PRs for `firestore-bigquery-export` #373 & `firestore-bigquery-change-tracker` #374. Two more  will follow for the `gen-schema-view` & `import` scripts that are part of the `firestore-bigquery-export` extension.

**Pull request closed without merging**